portfolio/src/main/webapp/script.js

```diff
@@ -20,5 +20,5 @@ async function sayHello() {
   const responseText = await serverResponse.text();
 
   const greeting = document.getElementById('hello-response');
-  greeting.innerText = responseText;
+  greeting.innerHTML = responseText;
```

Comment:
innerText is fine if you are changing the text.

Reply:
It did not function properly before this edit, I believe responseText is actually text formatted as html code.